tests/src/core/testqgsexpression.cpp

```diff
@@ -1575,6 +1575,35 @@ class TestQgsExpression: public QObject
       QTest::newRow( "geometries_to_array_multipoly" ) << "geom_to_wkt(array_get(geometries_to_array(geom_from_wkt('MULTIPOLYGON(((5 5,0 0,0 10,5 5)),((5 5,10 10,10 0,5 5)))')),1))" << false << QVariant( "Polygon ((5 5, 10 10, 10 0, 5 5))" );
       QTest::newRow( "geometries_to_array_emptygeom" ) << "array_length(geometries_to_array(geom_from_wkt('LINESTRING EMPTY')))" << false << QVariant( 1 );
       QTest::newRow( "geometries_to_array_nongeom" ) << "geometries_to_array('just a string')" << true << QVariant();
+      QTest::newRow( "dissolve_geom_multiPolypOINT" ) << "geom_to_wkt(dissolve_geometries(geom_from_wkt('MULTIPOLYGON(((0 0,5 0,5 5,0 0)),((5 5,5 10,10 10,10 5)))')))" << false << QVariant("MultiPolygon (((5 0, 0 0, 5 5, 5 0)),((5 10, 10 10, 10 5, 5 5, 5 10)))");
+      QTest::newRow( "dissolve_geom_multiPolyEdge" ) << "geom_to_wkt(dissolve_geometries(geom_from_wkt('MULTIPOLYGON(((0 0,5 0,5 5,0 0)),((5 5,5 0,10 10,10 5)))')))" << false << QVariant("");
```

Comment:
I'm not sure if this should be acceptable or if a safeguard should be added to prevent this.